Lib/pyclbr.py

```diff
@@ -284,6 +320,7 @@ def _readmodule(module, path, inpackage=None):
     f.close()
     return dict
 
+
```

Comment:
What I do when I care is bring up gitgui and it shows diffs for each uncommitted file (staged or not) in the branch.  I am not sure how to do the same with git diff.